docs/content/docs/usage/intl.md

```diff
@@ -0,0 +1,236 @@
+
+function compileLocale(locale, {messages, formats = {}}) {
+  return {locale, messages: flatten(messages), formats}
+}
+
+const LOCALES = {
+  "en": compileLocale("en", localeEN),
+  "fr": compileLocale("fr", localeFR),
+}
+
+export function setLocale(locale) {
+  window.localStorage.setItem("locale", locale)
+  return {type: SET_LOCALE, ...LOCALES[locale]}
+}
+```
+
+## Create intl reducer
+
+Create a `intl.js` file under a `reducers` folder
+
+```
+import {SET_LOCALE} from "constants/actions"
+
+const initialState = {
+  locale: null,
+  messages: null,
+  formats: null,
+}
+
+export default function intlReducer(state = initialState, action) {
+  if (action.type === SET_LOCALE) {
+    return {
+      locale: action.locale,
+      messages: action.messages,
+      formats: action.formats,
+    }
+  }
+
+  return state
+}
+```
+
+## Add intl reducer to your store
+
+```
+import {combineReducers} from "redux"
+...
+import * as phenomicReducers from "phenomic/lib/redux/modules"
+import intl from "reducers/intl"
+const store = createStore(
+  combineReducers({
+    ...phenomicReducers,
+    ...{
+      intl
+    },
+  }),
+  {...(typeof window !== "undefined") && window.__INITIAL_STATE__},
+  [
+    thunk,
+    createLogger({collapsed: true}),
+  ],
+)
+```
+
+## Get your translated content
+
+Add a metadata `locale` to your `.md` files
+
+```markdown
+---
+title: Your title
+locale: en
+---
+Your content
+```
+
+Then, use the `filter` option from `enhanceCollection` in your layouts to get your translated content :
+
+```javascript
+...
+class Homepage extends Component {
+  render() {
+    const {collection, store} = this.context
+    const locale = store.getState().intl.locale
+    const data = enhanceCollection(collection, {
+      filter: {locale},
+    })
+
+    return (
+      <Page>
+        // Render your data here
+      </Page>
+    )
+  }
+}
+```
+
+## Update your header
+
+Once you can have your translated content, we have to add two buttons in the header to change our locale.
+
+```javascript
+...
+import {setLocale} from "actions/intl"
+...
+class Header extends Component {
+  constructor() {
+    super()
+    this.updateLocale = this.updateLocale.bind(this)
+  }
+
+  updateLocale(locale) {
+    store.dispatch(setLocale(locale))
+  }
+
+  render() {
+    return (
+      <header>
+        <nav>
+          ...
+          <div onClick={() => this.updateLocale("en")}>{"en"}</div>
+          <div onClick={() => this.updateLocale("fr")}>{"fr"}</div>
+        </nav>
+      </header>
+    )
+  }
+}
+```
+
+## Initialize with right locale
+
+Now we need to initialize the site with the right locale.
+
+To do this, open your `scripts/phenomic.browser.js` and add :
+
+```javascript
+import {setLocale} from "actions/intl"
+const DEFAULT_LOCALE = "en"
+...
+let locale = window.localStorage.getItem("locale")
+
+if (locale !== null) {
+  store.dispatch(setLocale(locale))
+} else {
+  store.dispatch(setLocale(DEFAULT_LOCALE))
+}
+```
```

Comment:
I forgot to add the doc about "redirect to home page".

But yes, maybe we can imagine to test two urls and redirect to the same url with user locale if the url exists